pages/knowledge/screen-readers/desktop/browse-focus-modes/README.md

```diff
@@ -89,7 +89,7 @@ To make listening to the `Tab` key work again, a corresponding functionality nee
 
 Also, while leaving focus mode is always possible using the `Esc` key, this is not available in application mode as well. In the worst case, screen readers get trapped within such an element and will not be able to get out of it anymore. So again, this needs to be implemented using JavaScript - together with each and every other needed interactivity.
```

Comment:
Sounds a bit strange, the "not available ... as well" part. Should it not be something like "Also, while leaving focus mode is always possible using the `Esc` key, this option is not available in application mode." Or maybe even without the "Also", so "While leaving focus mode is always possible using the `Esc` key, this option is not available in application mode."

Reply:
Fixed. 👍

Reply:
The "as well" part should emphasise that this is one of the many things that are not available in application mode. I fixed it as you proposed, and added an "(like many other options)".

Reply:
All fine now - and it contains what you wanted to emphasise!